### ISSUE TITLE
Internationalization / translations

### ISSUE THREAD

**Issue by The-Compiler** [qutebrowser/qutebrowser#31]
If there are people willing to translate, maybe support for i18n should be added.

**Comment by moozzz** [comment]
Hi! I might help you with translation into German and Russian if you tell me what exactly has to be "internationalized" =) It could take a while though...

**Comment by The-Compiler** [comment]
Cool! I could do German myself :). But in general, this was just an idea, and I wonder if it's really worth the effort, given that people using such a browser are probably quite okay with English...

**Comment by moozzz** [comment]
Stimmt! Mir ist eben aufgefallen, dass Du aus der Schweiz kommst =D 

But perhaps you are right. People that came up with the idea to use your browser or any similar browsers (dwb, luakit, etc.) must have learnt English anyway =)

I've been using your browser for a couple of days and must say it's the most stable lightweight vim-like browser I have ever used. My previous default browser was luakit, and I really had troubles with it. Besides constantly crashing, it seemed to not release RAM after closing pages. So the RAM used by luakit just accumulated over time. Also, it had problems with saving sessions upon closing so that I had to manually correct the code for my local copy. (Sorry for off-topic writing)

I would be glad to contribute to your browser anyway, even if I am not a born programmer =)

**Comment by The-Compiler** [comment]
I'm closing this - probably not worth the effort looking at the small and technical userbase.

**Comment by The-Compiler** [comment]
Given that our userbase is a lot bigger and more international than it was five years ago, I think it's time to reconsider this. No immediate plans to add translation infrastructure (and I'm not even sure how well it'd work given that e.g. qutebrowser gets command descriptions from docstrings), but it certainly warrants a closer look.

**Comment by Oboro-Dev** [comment]
Hi. 
I am Japanese and I love your browser very much.
But I only regret that there are no other languages in this browser.
I'm not good at English and I'm a beginner in programming.
but I may be able to contribute in some way.

**Comment by jakubfabijan** [comment]
Yeah! I can translate to Polish, Czech and Esperanto.

**Comment by Oboro-Dev** [comment]
Awesome!
I can only translate from English to Japanese.
but I want to do my best.

**Comment by xanmodlouis** [comment]
I'm chinese, when i use the qutebrowser in my first time, I'm in love with it,
